### PR TITLE
MAINT: Update leveraged ETF list

### PR DESCRIPTION
**Pull request by LotannaEzenwa**
Updated with data using http://www.etf.com/etfanalytics/etf-finder

Checked against http://etfdb.com/etf-launch-center/

### CODE REVIEW

**Comment by LotannaEzenwa** [comment]
Any status on #754?
@jmccorriston

**Comment by jmccorriston** [comment]
Moving https://github.com/quantopian/zipline/pull/754 to this branch.

File: zipline/resources/security_lists/leveraged_etf_list/20020103/20160907/add [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+LABD
+GUSH
+DRIP
+LRET
+CHAU
+HOML
+GDXX
+GDJJ
+GDXS
+SMHD
```
Comment (llllllllll) [inline zipline/resources/security_lists/leveraged_etf_list/20020103/20160907/add]: add a newline then this looks functionally looks fine, no idea if these tickers are actually leveraged etfs.

Comment (jmccorriston) [inline zipline/resources/security_lists/leveraged_etf_list/20020103/20160907/add]: checked them against http://www.etf.com/etfanalytics/etf-finder

**Pull request merged**